autoscale/src/main/java/com/sequenceiq/periscope/service/DateService.java

```diff
@@ -19,7 +19,7 @@
 import com.sequenceiq.periscope.utils.TimeUtil;
 
 @Service
-public final class DateService {
+public class DateService {
```

Comment:
Accidental change?

Reply:
This was required for CronTimeEvaluatorTest but apparently recent mockito can support even final classes. Hence reverted this now.